config/overlays/dev/kustomization.yaml

```diff
@@ -1,3 +1,38 @@
 resources:
+  - namespace.yaml
```

Comment:
>I'm not sure if that was the intended outcome of make deploy-dev

Oh yeah I don't know what was the original motivation behind that, if that wasn't the original one I can revert the changes affecting `overlays/dev`

Reply:
I don't think it's a good idea to duplicate the contents of these overlays as it'll increase the difficulty of maintaining changes. For the `deploy-dev` installation, the `overlays/dev/kustomization.yaml` can call the `overlays/odh/kustomization.yaml` directly to avoid duplication

Reply:
yeah I couldn't find any reference to that `crd/external/kustomization.yaml`, at least not in this repo but I don't know if they could be referenced from other places outside this repo itself.